ts/screens/authentication/TestAuthenticationScreen.tsx

```diff
@@ -164,10 +203,19 @@ const TestAuthenticationScreen = (props: Props) => {
   );
 };
 
+const mapStateToProps = (state: GlobalState) => ({
+  maybeKeyTag: lollipopKeyTagSelector(state),
+  mixpanelEnabled: isMixpanelEnabled(state)
+});
+
 const mapDispatchToProps = (dispatch: Dispatch) => ({
   requestLogin: (passwordLogin: PasswordLogin) =>
     dispatch(testLoginRequest(passwordLogin)),
-  cleanUpLogin: () => dispatch(testLoginCleanUp())
+  cleanUpLogin: () => dispatch(testLoginCleanUp()),
+  dispatch
 });
 
-export default connect(undefined, mapDispatchToProps)(TestAuthenticationScreen);
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(TestAuthenticationScreen);
```

Comment:
What about using simply selectors and props?

Reply:
In this screen in addition to this change you suggested there would also be the `BaseScreenComponent` to replace. I was thinking of doing this in a separate PR, but if it's OK with you I can add to this PR, just tell me which solution you like best!

Reply:
If you wanna do it in a separate PR it's okay. But then use selector for `mixpanel` and the `keyTag` and `useIONavigation` to have `dispatch` without touching the as-is. What do. you think?

Reply:
I decided to change both the deprecated components and to delete mapDispatchToProps and mapStateToProps